src/classes/MyHandler.ts

```diff
@@ -457,10 +457,15 @@ export = class MyHandler extends Handler {
                         }
 
                         const item = SKU.fromString(sku);
+                        const buyPrice = match.buy.toValue(keyPrice.metal);
+                        const sellPrice = match.sell.toValue(keyPrice.metal);
+                        const minimumKeysDupeCheck = this.minimumKeysDupeCheck * keyPrice.toValue();
 
                         if (
-                            item.effect !== null &&
-                            match.buy.toValue(keyPrice.metal) > this.minimumKeysDupeCheck * keyPrice.toValue()
+                            (item.effect !== null || item.australium === true) && // check on only items with effects or australium
+                            buying && // check only items on their side
+                            (buyPrice > minimumKeysDupeCheck || sellPrice > minimumKeysDupeCheck)
+                            // if their side contains invalid_items, will use our side value
```

Comment:
Why only australiums and items with effects?

Reply:
Previously it was only `item.effect !== null`. What else that you think the best items to check for duped?

Reply:
I have changed it to check on all items that have > minimum keys to check